filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -1079,18 +1079,16 @@ void OpenGLDriver::createRenderTargetR(Handle<HwRenderTarget> rth,
     rt->gl.samples = samples;
     rt->targets = targets;
 
+    GLenum bufs[MRT::MAX_SUPPORTED_RENDER_TARGET_COUNT] = { GL_NONE };
+    const size_t maxDrawBuffers = getMaxDrawBuffers();
```

Comment:
Please add a comment about this in the PR description. It was a bug not to call glDrawBuffers for depth-only passes (used in post processing I think?)

Reply:
Added comment in the description.